### PR TITLE
[DPM-511] return deposit if player haven't made first action

### PR DESCRIPTION
**Pull request by alrxy**

### CODE REVIEW

File: sdk/include/game-contract-sdk/game_base.hpp [hunk 1]
```diff
@@ -140,9 +140,18 @@ class game : public eosio::contract {
 
     using session_table = eosio::multi_index<"session"_n, session_row>;
 
+    struct [[eosio::table("sesfirstact"), eosio::contract("game")]] session_first_action_row {
+        uint64_t ses_id;
+        bool acted; // true if player did first action
```
Comment (justefg) [inline sdk/include/game-contract-sdk/game_base.hpp]: can we add `acted` to the session table?

session is cleaned when the game finishes or `close` is called.

Comment (alrxy) [reply sdk/include/game-contract-sdk/game_base.hpp]: yes, we can, but in that case we should to carefully deploy contracts

File: sdk/include/game-contract-sdk/game_base.hpp [hunk 2]
```diff
@@ -417,8 +418,11 @@ class game : public eosio::contract {
         sessions.modify(session, get_self(), [&](auto& obj) {
             obj.last_update = eosio::current_time_point();
             obj.state = static_cast<uint8_t>(state::req_action);
+            if (!session.acted) {
+                obj.acted = true;
```
Comment (justefg) [inline sdk/include/game-contract-sdk/game_base.hpp:422]: what about just `obj.acted = true` ?

Comment (alrxy) [reply sdk/include/game-contract-sdk/game_base.hpp:422]: ok

**Review by justefg — APPROVED** [review]

**Pull request merged**